outputs.tf

```diff
@@ -22,4 +22,9 @@ output "crn" {
   value       = var.auto_initialization_using_recovery_crypto_units ? ibm_hpcs.hpcs_instance[0].crn : ibm_resource_instance.base_hpcs_instance[0].crn
 }
 
+output "endpoints" {
+  description = "HPCS instance extensions"
```

Comment:
We are nowhere using the term `extensions` except this. It is also reflected in ReadMe too. So better to change it as `endpoints` to maintain consistency.